### DISCUSSION TITLE
404 Not Found nginx/1.14.0

### DISCUSSION THREAD

**Discussion by Dougj2** [leokhoa/laragon#461] [Laragon's Articles & How-tos]
I am running Laragon 5.0.0.210523 and am trying to build a Joomla 4.2.9 site locally but every time I link to a page I get a 404 Not Found nginx/1/14/0/ error message. I assume I should be upgrading to the latest version of nginx so my questions is am I better to upgrade to the newest version of Laragon or should i upgrade nginx? Finding any info about upgrading anything other that the version of PHP running on Laragon seems difficult. How do I perform either task without loosing my current work?

D.

**Comment by Pen-y-Fan** [comment]
I'm not an expert in Joomla, so I checked their site and the requirements are low, even for the latest release:

- [Joomla requirements](https://downloads.joomla.org/technical-requirements)

I would recommend upgrading the Laragon 6, as support for PHP 8 is much better. See [How to update? #405](https://github.com/leokhoa/laragon/discussions/405) for my way.

Joomla 4.2.9 should run Nginx 1.14. I think the problem is local, maybe a problem with a windows update. 

- Have you tried rebooting?
- Installing the latest windows updates?
- Do other site work?
- You say 'every time I link to a page I get a 404...' is that some sort of plugin? Or do you mean all pages within the site?
